front/lib/models/assistant/conversation.ts

```diff
@@ -32,6 +32,8 @@ export class Conversation extends Model<
   declare title: string | null;
   declare visibility: CreationOptional<ConversationVisibility>;
 
+  declare groupIds: number[];
```

Comment:
Is it possible to type it more as a foreign key?

Reply:
we would need to have a join table, which I think is overkill. We can discuss and change it later if you think it's worth it

Reply:
Agreed it's not worth it